.github/workflows/main.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           - python: "3.8"
             image_tag: "3.19"
           - python: "3.9"
-            image_tag: "3.18"
```

Comment:
I know we say "tested on prev 5 versions" - but I'd rather see us test against 3.18, which is in use by a major stakeholder, than 3.19, which nobody that we know of has standardized on.

(We're also going to need to reconsider, when we move core to more-rapid releases...)

Reply:
Fair point.